internal/service/cluster/resource_cluster.go

```diff
@@ -351,6 +351,11 @@ func Resource() *schema.Resource {
 				Computed:    true,
 				Description: "Submit this field alongside your topology reconfiguration to request a new regional outage resistant topology",
 			},
+			"redact_client_log_data": {
+				Type:     schema.TypeBool,
+				Optional: true,
+				Computed: true,
```

Comment:
Can this be optional only? Or the API returns a value by default?

Reply:
yes, the API returns false if not set explicitly